OpenEdXMobile/src/main/java/org/edx/mobile/notifications/services/EdxFirebaseInstanceIdService.java

```diff
@@ -0,0 +1,14 @@
+package org.edx.mobile.notifications.services;
+import com.google.firebase.iid.FirebaseInstanceIdService;
+import com.google.firebase.iid.FirebaseInstanceId;
+import android.util.Log;
+
+public class EdxFirebaseInstanceIdService extends FirebaseInstanceIdService {
+    @Override
+    public void onTokenRefresh() {
+        // Get updated InstanceID token.
+        String refreshedToken = FirebaseInstanceId.getInstance().getToken();
+        Log.d(this.getClass().getSimpleName(), "Refreshed token: " + refreshedToken);
```

Comment:
Note: This code could be removed potentially.  It is useful for testing as you can target your individual device, but we are not relying on User device ID right now.